### ISSUE TITLE
[Bug] Fall damage kills "invincible" peds

### ISSUE THREAD

**Issue by Reldam88** [itsjustcurtis/MenyooSP#69]
The "Invincible" setting under Object Spooner>Ped Options doesn't protect NPCs from fall damage.

**Comment by itsjustcurtis** [comment]
Can you provide some steps to replicate for me please? How are you getting the peds in a position to take fall damage?

**Comment by Reldam88** [comment]
So, the easiest way to test this would probably be to select a ped in Spooner Mode and use the "Frozen In Place" function. Then use "Manual Placement" to raise them up into the air on the Z axis and then unfreeze them so they fall to their death.

I've had problems with my bodyguards following me on rooftops, especially up and down ladders and they just randomly fall to their deaths because R* NPC navigation is not the greatest. When NPCs lose their footing on steep slopes they can fall/roll to their death and they sometimes bail out of aircraft which obviously results in death.

**Comment by itsjustcurtis** [comment]
I've just tested this myself, appears to be working as expected. Using Spooner mode I marked the ped as frozen in place, set the z value to >100, unchecked frozen in place, and the peds health remains unchanged when landing. To confirm, I removed the invincibility option, and the peds health drops to zero on impact. 

Closing as unable to replicate, if you are able to get video of this happening then I'll re-open and investigate further.
